### ISSUE TITLE
install.sh: Please document usage of COMPANIONPI_BRANCH and COMPANION_BUILD

### ISSUE THREAD

**Issue by cprima** [bitfocus/companion-pi#18]
Please document the use of the lines
COMPANIONPI_BRANCH="${COMPANIONPI_BRANCH:-main}"
COMPANION_BUILD="${COMPANION_BUILD:-beta}"

As install.sh will call ./update.sh stable "$COMPANION_BUILD" the epxected value needs to be known.

When running ./update.sh stable the line
node "/usr/local/src/companionpi/update-prompt/main.js" $1
fails with 
No beta build was found!
so I assume I should kow how to set $COMPANION_BUILD in the first place, and continue to troubleshoot from there.

**Comment by Julusian** [comment]
It turns out that using `COMPANION_BUILD` was a bit broken, which I have fixed. 
There is some comments for those variables in the file, but I don't recommend their use so I won't document it outside of the file https://github.com/bitfocus/companion-pi/blob/main/install.sh#L20-L24

But I don't think my changes would fix the fact that it failed to find a build for you. No matter what I set `COMPANION_BUILD` to it would find and install `v3.0.0`. Are you running this on an arm install (not arm64)?

**Comment by cprima** [comment]
> But I don't think my changes would fix the fact that it failed to find a build for you. No matter what I set `COMPANION_BUILD` to it would find and install `v3.0.0`. Are you running this on an arm install (not arm64)?

Indeed I was running the script from companion-pi repo on a Raspberry Pi 4 with Raspbian in the 32 bit version.
I have now updated to a arm64 version (although of DietPi and not Raspbian), and *install.sh works* without a problem.

My root cause was: On that 32bit I must have updated Python3 to 3.11.x
*That leads down a rabbit hole*, which triggered me to investigate your install.sh and update.sh in the first place (and lead to me opening this issue).
My (limited) understanding of my root cause is: In Node v14.x.y npm is not compatible with Python3.11, I got a bunch of errors like "ValueError: invalid mode: 'rU' while trying to load binding.gyp" and "TS2688: Cannot find type definition file for node_modules". Stackoverflow is full of these questions.

Long story short: Once I used a current 64bit base OS all worked fine.

But hey, I now know every single line of your bash script, and I think I fully understood the v2 vs v3 logic. Quite a challenge to maintain!

**Comment by cprima** [comment]
Reading the packer file I now understand where these environment variables are coming from:
https://github.com/bitfocus/companion-pi/blob/main/companionpi.pkr.hcl
